docs/FAQ.rst

```diff
@@ -0,0 +1,276 @@
+the native operator. You can also become a contributor and make your work a part of
+the DALI code base - we would be more than happy to review and accept any PR with new
+functionality.
+
+Q: How to use DALI for inference?
+#################################
+A: You can easily employ DALI for inference together with the TRITON server. We developed
+a dedicated DALI backend so all the user needs to do is provide a description of the processing
+pipeline to the TRITON server, and add DALI into the model ensemble.
+
+Q: How much is the speedup of using DALI compared to loading using OpenCV? Especially for JPEG images.
+######################################################################################################
+A: DALI utilizes nvJPEG to accelerate JPEG decoding. It leads up to 2.5x speedup with
+NVIDIA A100 Ampere GPU - `see for details <ttps://developer.nvidia.com/blog/loading-data-fast-with-dali-and-new-jpeg-decoder-in-a100/>`_.
+In the case of other image formats, which decoding is not GPU accelerated, DALI uses either OpenCV
+(like for PNG) or dedicated library directly, like libtiff. In this case, the performance should
+be comparable.
+
+Q: Can you use DALI with DeepStream?
+####################################
+A: There is no reason why models trained with DALI can be used for inference with DeepStream.
+DeepStream mostly focuses on online video and audio inference pipelines, DALI can do this only
+for offline scenarios. There is no direct integration between these two solutions.
+
+Q: How to control the number of frames in a video reader in DALI?
+#################################################################
+A: It can be controlled by the `sequence_length` argument.
+
+Q: Does DALI volumetric 3D data processing can work for the ultrasound?
+#######################################################################
+A: DALI doesn't support any domain-specific data formats like NIfTI, but in most cases, the data
+is first processed offline and converted to a more data processing friendly format like NumPy
+arrays (including initial normalization and conversion). If the data is available in DALI-supported
+format there is no reason why it cannot be processed no matter if it is an ultrasound or CT scan.
+
+Q: How to debug the DALI pipeline?
+##################################
+A: Just recently DALI added `an eager debug mode <../examples/general/debug_mode.html>`_ so
+the output of each operator can be instantly evaluated and andy python code can be added inside
+the pipeline for the prototyping purpose.
+
+Q: Can I access the contents of intermediate data nodes in the pipeline?
+########################################################################
+A: In the pipeline mode it is not possible, however, thanks to the recently introduced
+`debug mode <../examples/general/debug_mode.html>`_` it can be done for prototyping
+and debugging purposes.
```

Comment:
```suggestion
A: In the pipeline mode it is not possible, however, thanks to the recently introduced
`debug mode <../examples/general/debug_mode.html>`_` it can be done. For performance
reasons, this feature is intended only for debugging and prototyping.
```

Reply:
Fixed